components/cards/CardOne.tsx

```diff
@@ -0,0 +1,44 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+
+import IconContainer from './IconContainer';
+import { ReactNode } from 'react';
+import Button from '@mui/material/Button';
+
+
+type CardOneProps = {
+    title?: string,
+    description: string,
+    icon: ReactNode,
+    buttonText?: string,
+    buttonLink?: string
+}
+
+const CardOne = ({title, description, icon, buttonText, buttonLink}: CardOneProps) => {
+    return (
+        <Card sx={{ 
```

Comment:
I didn't know about this way of using breakpoints with MUI. Thanks for showing me!

Reply:
Yeah I found out in the MUI docs! We can also modify the breakpoints in the theme file if needed 👍